Examples/Python/src/Base.cpp

```diff
@@ -267,4 +269,29 @@ void addAlgebra(Acts::Python::Context& ctx) {
            [](const Acts::Vector4& self, Eigen::Index i) { return self[i]; });
 }
 
+void addBinning(Context& ctx) {
+  auto& m = ctx.get("main");
+  auto binning = m.def_submodule("Binning", "");
+
+  auto binningValue = py::enum_<Acts::BinningValue>(binning, "AxisValue")
+                          .value("x", Acts::BinningValue::binX)
+                          .value("y", Acts::BinningValue::binY)
+                          .value("z", Acts::BinningValue::binZ)
+                          .value("r", Acts::BinningValue::binR)
+                          .value("phi", Acts::BinningValue::binPhi)
+                          .export_values();
+
+  auto boundaryType =
+      py::enum_<Acts::detail::AxisBoundaryType>(binning, "AxisBoundary")
```

Comment:
I guess it makes sense to keep the naming consistent? Maybe we need to promote `AxisBoundaryType` out of `detail` anyway.

Reply:
I agree with the `detail` change, I will rename as well. The detail change I will do at another stage.